src/models/converters/transactions/tests/mod.rs

```diff
@@ -1,4 +1,45 @@
+use crate::models::converters::transactions::data_size;
+
 mod data_size_calculation;
 mod summary;
 mod details;
-mod transfer_type_checks;
```

Comment:
Aren't these the same tests as above?

Also not sure if I would add tests the mod.rs

Reply:
I removed the tests from mod. `transfer_type_checks` were the methods checking `is_settings_change` , `is_ether_transfer`, `is_erc20_transfer` and `is_erc721_transfer` which in the current implementation don't exist. I will add more tests trying to cover the types, but yeah, there is also fewer methods performing the checks.